ci-operator/step-registry/openstack/scale/masters/openstack-scale-masters-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+info() {
+	printf '%s: %s\n' "$(date --utc +%Y-%m-%dT%H:%M:%SZ)" "$*"
+}
+
+if [ -z "$SCALE_MASTERS_FLAVOR_ALTERNATE" ]; then
+	info "All defined environment variables for the step are empty. Exiting."
+	exit 0
+fi
+
+declare patch=''
+
+info "Detected the environment variable SCALE_MASTERS_FLAVOR_ALTERNATE set to '$SCALE_MASTERS_FLAVOR_ALTERNATE'. Patching the CPMS..."
+
+patch="$(printf '{"spec":{"template":{"machines_v1beta1_machine_openshift_io":{"spec":{"providerSpec":{"value":{"flavor":"%s"}}}}}}}' "$(<"${SHARED_DIR}/OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE")")"
+
+oc patch controlplanemachineset.machine.openshift.io --type=merge --namespace openshift-machine-api cluster -p "$patch"
+
+info 'Waiting one minute for the CPMS operator to pick up the edit...'
+sleep 60
+
+info 'Waiting for the rollout to complete...'
+# shellcheck disable=SC2046
+oc wait --timeout=90m --for=condition=Progressing=false controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster -o template='{{.metadata.name}} is ready\n'
```

Comment:
This `\n` is not being interpreted as EOL

Reply:
I've updated the line, check it out if it's ok now.